libs/s3/iam_test_lib.py

```diff
@@ -143,19 +125,14 @@ def delete_access_key(
         :return: (Boolean, response).
         """
         try:
-            LOGGER.info(
-                "Deleting %s user access key %s.",
-                user_name,
-                access_key_id)
+            LOGGER.info("Deleting %s user access key %s.", user_name, access_key_id)
             response = poll(super().delete_access_key, user_name, access_key_id)
             LOGGER.info(response)
             # Adding sleep in ms due to ldap sync issue EOS-25140
             time.sleep(S3_CFG["access_key_delay"])
         except (ClientError, Exception) as error:
-            LOGGER.error("Error in %s: %s",
-                         IamTestLib.delete_access_key.__name__,
-                         error)
-            raise CTException(err.S3_CLIENT_ERROR, error.args[0])
+            LOGGER.error("Error in %s: %s", IamTestLib.delete_access_key.__name__, error)
+            raise CTException(err.S3_CLIENT_ERROR, error.args[0]) from error
```

Comment:
You re trying to re raise client error even for exception.

Reply:
reverted